test/functional/wallet_resendwallettransactions.py

```diff
@@ -10,7 +10,7 @@
 class ResendWalletTransactionsTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
-        self.extra_args = [['--walletbroadcast=false']]
+        self.extra_args = [['-walletbroadcast=0']]
```

Comment:
Feels like this should also be in #12713.

Reply:
This has been moved to the other commit.